ci/release-qualification/pipeline.template.yml

```diff
@@ -99,6 +99,18 @@ steps:
           composition: zippy
           args: [--scenario=DebeziumPostgres, --actions=1000000]
 
+  - id: zippy-backup-and-restore-large
```

Comment:
Is it stable enough for release qualification testing?

Reply:
I am going to do one or more runs before I can say that. I am using the Release Qualification pipeline as a "various very long" tests pipeline. But maybe it makes sense to have a "Weekly" pipeline instead where stuff like that could run.